### PR TITLE
Couple of fixes for BlackMagicProbe

### PR DESCRIPTION
**Pull request by kapacuk**
- Added a missing import to `modm_tools/bmp.py`
- Changed `guess_serial_port()` to use `/dev/ttyBmpGdb` when guessing a BMP port

### CODE REVIEW

File: tools/modm_tools/bmp.py
Comment (rleh) [inline tools/modm_tools/bmp.py:1]: Nice catch to change the file permissions to _755_ 👍🏽

File: tools/modm_tools/utils.py
Comment (rleh) [inline tools/modm_tools/utils.py:1]: Since this script can not be used directly, I would prefer to keep the _644_ permissions here.

**Review by rleh — COMMENTED** [review]
Hi @kapacuk and welcome!

Thank you very much for your contribution. Looks fine, just two minor comments (see below).

File: tools/modm_tools/utils.py
Comment (kapacuk) [reply tools/modm_tools/utils.py:1]: You're right, I reverted it back to 644

**Review by rleh — APPROVED** [review]

**Review by salkinium — APPROVED** [review]
Awesome, thanks! (Please squash into one commit.)

**Comment by salkinium** [comment]
Never mind, just remembered I can squash and merge from the GitHub UI. 🤦‍♂️

**Pull request merged**